src/rest_api.c

```diff
@@ -89,18 +166,34 @@ static int handler(struct mg_connection * conn, void * ignored)
 
 int auth_handler(struct mg_connection * conn, void * cbdata)
 {
-    int authorized = 1;
+    int authorized = 0;
+
+    point_t *public_key = (point_t *)cbdata;
+    if(public_key != NULL){
+        fprintf(stderr, "Public key is received: %s\n", public_key);
+    }
 
     char * auth_token = mg_get_header(conn, "Authorization");
     if(auth_token != NULL){
-        ///////////////////////////////////////////////////////
-        // TODO: Validate JWT TOKEN HERE
-        ///////////////////////////////////////////////////////
+        //Validate JWT TOKEN 
```

Comment:
comment style (two spaces between the code line and //, and one space after //) 

i++;  // comment